net-misc/asterisk/asterisk-18.24.1.ebuild

```diff
@@ -330,13 +328,23 @@ src_install() {
 }
 
 pkg_postinst() {
-	if [ -z "${REPLACING_VERSIONS}" ]; then
+	if [[ -z "${REPLACING_VERSIONS}" ]]; then
 		elog "Asterisk Wiki: https://wiki.asterisk.org/wiki/"
 		elog "Gentoo VoIP IRC Channel: #gentoo-voip @ irc.libera.chat"
-	elif [ "$(ver_cut 1 "${REPLACING_VERSIONS}")" != "$(ver_cut 1)" ]; then
-		elog "You are updating from Asterisk $(ver_cut 1 "${REPLACING_VERSIONS}") upgrade document:"
-		elog "https://wiki.asterisk.org/wiki/display/AST/Upgrading+to+Asterisk+$(ver_cut 1)"
-		elog "Gentoo VoIP IRC Channel: #gentoo-voip @ irc.libera.chat"
+	else
+		local my_replacing=() x
+		for x in ${REPLACING_VERSIONS}; do
+			[[ "$(ver_cut 1 "${REPLACING_VERSIONS}")" != "$(ver_cut 1)" ]] && \
+				my_replacing+=( $(ver_cut 1 "${REPLACING_VERSIONS}") )
+		done
+		if [[ "${#my_replacing}" -gt 0 ]]; then
+			my_replacing="$(printf "%d, " "${my_replacing[@]}")"
+			my_replacing="${my_replacing%%, }"
+			[[ "${my_replacing}" = *", "* ]] && my_replacing="${my_replacing%, *} or ${my_replacing##*, }"
```

Comment:
This really looks like an overkill.

Reply:
Yea, not to mention if you've somehow managed to get two version 18s installed and now upgrading to 20 you'll get 18 or 18 ... fixing the grammar was semi important to me as it will be confusing otherwise, the same number twice too, but alas, that'll be less confusing than your upgrading from 16, 18, 20 kind of messages.  So I think whilst it may be overkill let's leave this one if you don't mind.

Reply:
Then at least wrap the line.